simpleLooping/test_incrementBinaryList.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from incrementBinaryList import incrementBinaryList
+
+class TestIncrementBinaryList(unittest.TestCase):
+    def test_overFlowListExpansion(self):
+        self.assertEqual(incrementBinaryList([1, 1, 1, 1]), ([0, 0, 0, 0, 1]))
+
+    def test_regularBinaryList(self):
```

Comment:
Hi, I find this name via ChatGPT, how do you think “BinaryIncrementNonOverflowInput”.